main.js

```diff
@@ -0,0 +1,92 @@
+var cities = [];
```

Comment:
Will `cities` ever have weather data for more than just one city? It probably shouldn't be an array and perhaps you could call it something besides `cities`